packages/redeem-failed-retryable/scripts/exec-redeem.js

```diff
@@ -46,9 +50,10 @@ module.exports = async txnHash => {
   /**
    * We use the redeem() method from Arbitrum SDK to manually redeem our ticket
    */
-  await message.redeem()
+  const l2Tx = await message.redeem()
+  const rec = await l2Tx.waitForRedeem()
   console.log(
     'The L2 side of your transaction is now execeuted 🥳 :',
-    message.l2TxHash
+    await rec.transactionHash
```

Comment:
do we need to await here?

Reply:
I think it's not bad to have it, in case user wants to check the L2 transaction on explorer etc.